x-pack/plugin/ml/qa/native-multi-node-tests/src/javaRestTest/java/org/elasticsearch/xpack/ml/integration/DetectionRulesIT.java

```diff
@@ -299,6 +304,62 @@ public void testScopeAndCondition() throws IOException {
         assertThat(records.get(0).getOverFieldValue(), equalTo("222.222.222.222"));
     }
 
+    public void testForceTimeShiftAction() throws Exception {
+        // The test ensures that the force time shift action works as expected.
+
+        long timeShiftAmount = 3600L;
+        long timestampStart = 1491004800000L;
```

Comment:
```suggestion
        long timestampStartMillis= 1491004800000L;
```

Nit I was confused about which are epoch millis and which are epoch seconds. `timeShiftTimestamp` could be renamed `timeShiftTimestampSecs` too